TYPO3.Media/Resources/Private/Templates/Asset/Index.html

```diff
@@ -40,10 +40,16 @@
 			</span>
 			<ul class="neos-dropdown-menu neos-pull-right" role="menu">
 				<li>
-					<f:link.action action="index" title="Sort by last modified" arguments="{sort: 'Modified'}" addQueryString="TRUE" class="{f:if(condition: '{sort} == \'Modified\'', then: 'neos-active')}"><i class="icon-sort-by-order"></i> Last modified</f:link.action>
+					<f:link.action action="index" title="Sort by last modified" arguments="{sort: 'Modified-ASC'}" addQueryString="TRUE" class="{f:if(condition: '{sort} == \'Modified-ASC\'', then: 'neos-active')}"><i class="icon-sort-by-order"></i> Last modified</f:link.action>
```

Comment:
good point, although that should probably be done in a separate change since this cannot go into 2.0